bats.yaml

```diff
@@ -28,9 +26,27 @@ pipeline:
       expected-commit: 5da66876b8b619235aee1eb3e54954eaca88059b
 
   - runs: |
-      ./install.sh  "${{targets.contextdir}}/usr"
+      ./install.sh "${{targets.contextdir}}/usr"
+      mkdir -p "${{targets.contextdir}}/tmp/"
+      cp -r ./docker "${{targets.contextdir}}/tmp/docker"
+      "${{targets.contextdir}}/tmp/docker/install_libs.sh" support 0.3.0
+      "${{targets.contextdir}}/tmp/docker/install_libs.sh" file 0.4.0
+      "${{targets.contextdir}}/tmp/docker/install_libs.sh" assert 2.1.0
+      "${{targets.contextdir}}/tmp/docker/install_libs.sh" detik 1.3.1
+      mkdir -p "${{targets.contextdir}}/usr/local/bin"
+      ln -s /opt/bats/bin/bats "${{targets.contextdir}}/usr/local/bin/bats"
```

Comment:
we can consider moving this to compat package given most of `/opt` fs is within the compat package. 💭

Reply:
Great suggestion!